[2025-05-19T17:13:51.698+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-19T17:13:51.767+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hh_vacancies_parser.parse_and_save manual__2025-05-19T17:13:48.118227+00:00 [queued]>
[2025-05-19T17:13:51.801+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hh_vacancies_parser.parse_and_save manual__2025-05-19T17:13:48.118227+00:00 [queued]>
[2025-05-19T17:13:51.803+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-05-19T17:13:51.849+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): parse_and_save> on 2025-05-19 17:13:48.118227+00:00
[2025-05-19T17:13:51.861+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=393) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-19T17:13:51.864+0000] {standard_task_runner.py:72} INFO - Started process 394 to run task
[2025-05-19T17:13:51.864+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'hh_vacancies_parser', 'parse_and_save', 'manual__2025-05-19T17:13:48.118227+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/pars_hh_***.py', '--cfg-path', '/tmp/tmp9pczp9o0']
[2025-05-19T17:13:51.866+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask parse_and_save
[2025-05-19T17:13:52.017+0000] {task_command.py:467} INFO - Running <TaskInstance: hh_vacancies_parser.parse_and_save manual__2025-05-19T17:13:48.118227+00:00 [running]> on host db4eb1a3da04
[2025-05-19T17:13:52.311+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hh_vacancies_parser' AIRFLOW_CTX_TASK_ID='parse_and_save' AIRFLOW_CTX_EXECUTION_DATE='2025-05-19T17:13:48.118227+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-19T17:13:48.118227+00:00'
[2025-05-19T17:13:52.331+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-19T17:13:52.898+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-19T17:13:52.918+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-19T17:13:52.919+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=hh_vacancies_parser, task_id=parse_and_save, run_id=manual__2025-05-19T17:13:48.118227+00:00, execution_date=20250519T171348, start_date=20250519T171351, end_date=20250519T171352
[2025-05-19T17:13:52.978+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-19T17:13:53.017+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-19T17:13:53.020+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
